### PR TITLE
Cleans up Native Module checks

### PR DESCRIPTION
**Pull request by andresesfm**
Removed a lot of code that is redundant now that we have better split the Android Native modules

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -203,6 +203,36 @@ Linking the package manually is not required anymore with [Autolinking](https://
     }
   ```
 
+  Ypu have two options depending on the stores you support:
```
Comment (hyochan) [inline README.md:206]: I see the `typo` here `Ypu`.

File: src/iap.ts [hunk 2]
```diff
@@ -29,65 +29,43 @@ const ANDROID_ITEM_TYPE_SUBSCRIPTION = 'subs';
...
-let iapFallbackInstallSourceAndroid = InstallSourceAndroid.GOOGLE_PLAY;
-
-export function setFallbackInstallSourceAndroid(
-  installSourceAndroid: InstallSourceAndroid,
-): void {
-  iapFallbackInstallSourceAndroid = installSourceAndroid;
-}
 
+/**
+ * This function will be made internal (not exported) in future versions.
```
Comment (hyochan) [inline src/iap.ts:34]: Could you kindly point to the related doc for this? The `deprecation` notice.

**Review by hyochan — CHANGES_REQUESTED** [review]
Thanks for the updates :pray:. However I am seeing lots of linting errors so far.

image

Can you please `pull` your sourcecode and run `yarn lint`? I've just found that the `lint` script should be updated although `ide` can grab it.

File: src/iap.ts (hunk 2 above)
Comment (andresesfm) [reply src/iap.ts:34]: https://github.com/dooboolab/react-native-iap/pull/1374/files#diff-d710196e59f78f9faa00b02fa321ef3cf62914e393feecb47a9b6d6f1f8215b4R32

Comment (hyochan) [reply src/iap.ts:34]: Ok looks like this `deprecation` notice will be handled by some other experts like you :laughing:

**Review by hyochan — APPROVED** [review]
Again! Thanks for everything @andresesfm.

Please submit an expense of `$80` for a small gift in https://opencollective.com/react-native-iap which includes your previous `PR`.

**Comment by hyochan** [comment]
@andresesfm Do yo have any idea of `CI` failing in `android_example`?

**Pull request merged**